googleapis/google-cloud-node
# migrate code from googleapis/nodejs-network-management

- feat: initial stub of library
- feat: add initial samples
- feat: add initial samples and tests (#1)
- chore(deps): update dependency sinon to v11 (#5)
- chore: release 1.0.0 (#6)
- chore(nodejs): update client ref docs link in metadata (#9)
- fix: failing tests for LRO check methods (#8)
- chore: release 1.0.1 (#10)
- build(nodejs): correct artifact name for npm (#11)
- fix(deps): google-gax v2.24.1 (#13)
- chore: release 1.0.2 (#14)
- feat: turns on self-signed JWT feature flag (#15)
- chore: release 1.1.0 (#16)
- chore: disable renovate dependency dashboard (#1194) (#18)
- fix(build): switch primary branch to main (#19)
- chore: release 1.1.1 (#20)
- chore: relocate owl bot post processor (#23)
- fix: upgrade module to GA (#22)
- chore: release 1.1.2 (#24)
- build(node): update deps used during postprocessing (#1243) (#29)
- chore(deps): update dependency @types/node to v16 (#30)
- chore(deps): update dependency sinon to v12 (#31)
- docs(samples): add example tags to generated samples (#32)
- build: add generated samples to .eslintignore (#33)
- docs(node): support "stable"/"preview" release level (#1312) (#36)
- chore: add api_shortname and library_type to repo metadata (#35)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#38)
- test(nodejs): remove 15 add 16 (#1322) (#40)
- build: update copyright year to 2022 (#42)
- chore: update v2.12.0 gapic-generator-typescript (#44)
- chore(deps): update dependency sinon to v13 (#46)
- docs(nodejs): version support policy edits (#1346) (#48)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#49)
- chore: update v2.14.2 gapic-generator-typescript (#53)
- build(node): update client library version in samples metadata (#1356) (#60)
- chore: remove unused imports (#61)
- chore(deps): update dependency @types/mocha to v9 (#62)
- chore(deps): update dependency sinon to v14 (#65)
- build!: update library to use Node 12 (#67)
- chore(main): release 2.0.0 (#68)
- chore(deps): update dependency jsdoc-fresh to v2 (#71)
- chore(deps): update dependency jsdoc-region-tag to v2 (#72)
- feat: support regapic LRO
- chore(main): release 2.1.0 (#74)
- feat: add new abort cause and new route next hop type (#76)
- chore(deps): update dependency linkinator to v4 (#75)
- chore(main): release 2.2.0 (#77)
- feat: add new deliver targets and drop causes for Private Service Connect (#78)
- chore: remove unused proto imports (#80)
- fix: better support for fallback mode (#81)
- fix: change import long to require (#82)
- fix: do not import the whole google-gax from proto JS (#1553) (#84)
- fix: allow passing gax instance to client constructor (#85)
- chore(main): release 2.3.0 (#79)
- fix: preserve default values in x-goog-request-params header (#86)
- test: use fully qualified request type name in tests (#89)
- fix(deps): use google-gax v3.5.2 (#93)
- chore(deps): update dependency @types/node to v18 (#91)
- chore(deps): update dependency jsdoc to v4 (#95)
- fix: regenerated protos JS and TS definitions (#96)
- chore(main): release 2.3.1 (#87)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
